### DISCUSSION TITLE
Limit plugins loaded on startup?

### DISCUSSION THREAD

**Discussion by jeff303** [prestodb/presto#16124] [Q&A] [answered]
I'm using Presto inside a Docker container based on `starburstdata/presto:350-e.7`.  What I notice is that bringing up the (single node) cluster takes a very long time, because it's loading so many plugins.  Is there a way to limit the plugins that are included, in order to reduce the startup time?  I believe the only engine I actually need for my scenario is the memory one.  I don't need Postgres, Mongo, Redshift, Cassandra, etc. (which are seen getting loaded in the logs).  Thanks for any pointers.

**Comment by bitsondatadev** [comment]
Hey Jeff, 

Starburst is actually based on the Trino ([formerly PrestoSQL](https://trino.io/blog/2020/12/27/announcing-trino.html)). Here's the [Starburst docs containing all the versions](https://docs.starburst.io/latest/release.html). Any Starburst version greater than [354-e](https://docs.starburst.io/latest/release/release-354-e.html) has the Trino branding.

All that to say, you should ask this on the [Trino discussion board](https://github.com/trinodb/trino/discussions) or Trino slack channel.

**Comment by rohanpednekar** [comment, marked as answer]
Hi @jeff303 You can use [Docker container](https://hub.docker.com/r/ahanaio/prestodb) based on PrestoDB 

`docker pull ahanaio/prestodb
`
 Feel free to get in contact with me if you need any additional information.
Thanks for checking in!
